dev/integration_tests/web_e2e_tests/test_driver/capabilities_integration_canvaskit.dart

```diff
@@ -0,0 +1,16 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Capabilities integration test', (WidgetTester tester) async {
```

Comment:
More precise test descriptions are more useful. Suggestion: "isCanvasKit returns true in CanvasKit mode"